packages/vscode-apollo/src/statusBar.ts

```diff
@@ -1,36 +1,37 @@
-import {
-  window,
-  StatusBarItem,
-  StatusBarAlignment,
-  ExtensionContext,
-  commands
-} from "vscode";
-import { LanguageClient } from "vscode-languageclient";
+import { window, StatusBarAlignment } from "vscode";
 
 export default class ApolloStatusBar {
-  private statusBarItem: StatusBarItem = window.createStatusBarItem(
-    StatusBarAlignment.Right
-  );
+  private _statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right);
 
-  constructor(context: ExtensionContext, client: LanguageClient) {
-    this.statusBarItem.text = "Apollo GraphQL $(rss)";
-    this.statusBarItem.show();
+  static loadingStateText = "Apollo GraphQL $(rss)";
+  static loadedStateText = "ApolloGraphQL $(rocket)";
+
+  constructor() {
+    this._statusBarItem.text = ApolloStatusBar.loadingStateText;
```

Comment:
I find it a little odd to refer to the `ApolloStatusBar` here. I'd say either make them static constants like `LOADING_STATE_TEXT` at the top of the file, or reference the class statics as `this.loadintStateText`

Reply:
I thought the same with the class static and `this.loadingStateText` first, but it's a TS error!
![image](https://user-images.githubusercontent.com/29644393/50714409-62487800-102d-11e9-9826-7705a9b36f13.png)

The purpose for this is so that if the text changes, the value is updated for the test as well. I can either leave as is or go with the first option and export those constants.